src/coprocessor/dag/executor/topn_heap.rs

```diff
@@ -173,6 +173,9 @@ impl PartialOrd for SortRow {
     }
 }
 
+unsafe impl Send for TopNHeap {}
```

Comment:
Why does it need to explicitly impl `Send`?

Reply:
We will run Executor in `CpuPool`, which needs this.